src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
I've added that skip back in, and linked to this thread in a comment by way of helping to explain.